ts/Core/Tooltip.ts

```diff
@@ -1370,51 +1380,69 @@ class Tooltip {
         } else {
             // update text
             if (tooltip.split) {
-                this.renderSplit(text as any, splat(pointOrPoints));
+                this.renderSplit(text as any, points);
             } else {
-                const label = tooltip.getLabel();
+                let checkX = x;
+                let checkY = y;
 
-                // Prevent the tooltip from flowing over the chart box (#6659)
-                if (!(options.style as any).width || styledMode) {
-                    label.css({
-                        width: this.chart.spacingBox.width + 'px'
-                    });
+                if (mouseEvent && currentSeries.is('xrange')) {
+                    checkX = mouseEvent.chartX - chart.plotLeft;
+                    checkY = mouseEvent.chartY - chart.plotTop;
                 }
 
-                label.attr({
-                    text: text && (text as any).join ?
-                        (text as any).join('') :
-                        text
-                });
-
-                // Set the stroke color of the box to reflect the point
-                label.removeClass(/highcharts-color-[\d]+/g)
-                    .addClass(
-                        'highcharts-color-' +
-                        pick(
-                            (point as any).colorIndex,
-                            currentSeries.colorIndex
-                        )
-                    );
+                if (
+                    chart.polar ||
+                    this.isInside(checkX, checkY, currentSeries, !point.tooltipPos) ||
+                    currentSeries.is('windbarb') // Windbarb is outside plot
```

Comment:
So `directTouch` just seems to be a flag set once on the series prototype and never changed, there is also `Pointer.isDirectTouch` but thats going to break the scrollablePlotArea part of this.

Reply:
Okay, here's a case that is not a windbarb: https://jsfiddle.net/highcharts/8q2vyzfk/ . Hover the top column. The thing is that this should have a tooltip because it is directly touched. In v9.0.1 it works.

Reply:
Still there's the `currentSeries.is('windbarb')` check...